ga4gh/backend.py

```diff
@@ -141,6 +161,37 @@ def startProfile(self):
     def endProfile(self):
         pass
 
+    def callSetsGenerator(self, request):
```

Comment:
It would be a bit neater to keep the `XGenerator` methods group together, and not have the startProfile and endProfile method sandwiched in between.